app/src/main/java/com/futurice/freesound/feature/common/ui/adapter/RecyclerViewAdapter.java

```diff
@@ -38,10 +38,10 @@
 /**
  * Implementation of {@link RecyclerView.Adapter} for {@link DisplayableItem}.
  */
-public final class RecyclerViewAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+public final class RecyclerViewAdapter<T> extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```

Comment:
The whole idea of DisplayableItem is to reduce boiler plate, so you wouldn't need an interface that all the elements in the list would have to extend, rather have them wrapped in some object, and unwrapped and casted later.

When you specify here <T> I think the whole idea of having a DisplayableItem loses it's intention and is meaningless, could be removed.